### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by LuigiMenez — DISMISSED** [review]
It's good for me.

File: src/components/Carousel/index.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import { useState } from 'react/cjs/react.development';
```
Comment (LoicBrassart) [inline src/components/Carousel/index.jsx]: ... Nope ?

File: src/components/Carousel/index.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import './style.css';
+import { FaArrowCircleRight, FaArrowCircleLeft } from 'react-icons/fa';
+
+export default function Cocktail({ instructionString }) {
+  const [instructions, setInstruction] = useState([]);
+  const [activSlice, setActivSlice] = useState(0);
+
+  const nextSlice = () => {
```
Comment (LoicBrassart) [inline src/components/Carousel/index.jsx]: nextSli**d**e ? prevSli**d**e ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A few Evils to fix

File: src/components/Carousel/index.jsx (hunk 1 above)
Comment (PierreGoe) [reply src/components/Carousel/index.jsx]: done

File: src/components/Carousel/index.jsx (hunk 2 above)
Comment (PierreGoe) [reply src/components/Carousel/index.jsx]: done

**Comment by LoicBrassart** [comment]
No SM validation

**Review by LuigiMenez — DISMISSED** [review]
Good for me

**Comment by LoicBrassart** [comment]
No SM validation

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: src/components/CocktailDetail/index.jsx [hunk 3]
```diff
@@ -3,6 +3,8 @@ import axios from 'axios';
 import './style.css';
 import { useParams } from 'react-router-dom';
 import Carousel from '../Carousel';
+import PropTypes from 'prop-types';
+
```
Comment (LoicBrassart) [inline src/components/CocktailDetail/index.jsx]: (Almost sure that Prettier/ESLint forbid you having 2 successive empty lines... I iz suspicious, have you set it up ?)

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**